oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/oidc/authentication/OidcIdTokenValidator.java

```diff
@@ -137,9 +136,15 @@ public void setClockSkew(Duration clockSkew) {
 	}
 
 	private static OAuth2Error invalidIdToken(Map<String, Object> invalidClaims) {
-		String claimsDetail = invalidClaims.entrySet().stream()
-				.map(it -> it.getKey() + " (" + it.getValue() + ")")
-				.collect(Collectors.joining(", "));
+		StringBuilder claimsDetail = new StringBuilder();
```

Comment:
Actually, since this is simply going to an error message, let's simply rely on the `toString` method. The code will be easier to read and to maintain:

```java
return new OAuth2Error("invalid_id_token",
    "The ID Token contains invalid claims: " + invalidClaims,
// ...
```

Also, your unit test will also need an update.

Reply:
change and update a unit test